src/tutorials/external-resources.mdx

```diff
@@ -138,91 +138,103 @@ As you configure your external resources in this section, keep in mind the follo
 
 <div style={{ display: 'flex', alignItem: 'center', justifyContent: 'center' }}>
   <table>
-    <tr>
+    <tr style={{ backgroundColor: 'var(--ifm-menu-color-background-active)' }}>
```

Comment:
I'm not sure how we're handling the styles in these cases, so I opted to add them inline.  Let me know if there's a better way to do so.

Reply:
I thought of defining a new CSS variable, but since it would have the same color and is used only here, maybe it would add unnecessary complexity, so I finally used the existing `--ifm-menu-color-background-active` one.